.github/workflows/test.yml

```diff
@@ -1,25 +1,28 @@
-# This workflow will install Python dependencies, run tests and lint with a single version of Python
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+# This workflow will install Python dependencies, run tests and lint with a
+# single version of Python. For more information see:
+# https://help.github.com/actions/language-and-framework-guides\
+# /using-python-with-github-actions
 
 name: Test OpenUpgrade migration
 
 on:
   push:
-    branches: ["14.0*"]
+    branches: ["14.0"]
```

Comment:
Doing this change PRs will be executed?

Reply:
Mmm, I must admit I changed this because it was one of the suggestions on the previous PR. But I see that it was updated in 13.0 related to runbot https://github.com/OCA/OpenUpgrade/commit/5801882f6e696d20011e11829dc97d8f441f7cc7.
I've put it back.

Reply:
Oh, that was the reason. I remember it was there due to something, but I didn't remember the exact one.